src/lib/optimizer/strategy/dependent_group_by_reduction_rule.hpp

```diff
@@ -12,15 +12,15 @@ struct TableConstraintDefinition;
 class StoredTableNode;
 
 /**
- * In SQL, the returned columns of an aggregate are either "group-by columns" or "aggregate columns, e.g., SUM(a)".
- * As an example, in TPC-H Q10, the aggregate on customer includes both c_custkey as well as c_name (amonst others) in
+ * In SQL, the returned columns of an aggregate are either "group-by columns" or "aggregate columns", e.g., SUM(a). As
+ * an example, in TPC-H Q10, the aggregate on customer includes both c_custkey as well as c_name (amonst others) in
  * the group-by columns.
  * Note how c_name is included as a group-by column, even though the groups are already exhaustively defined by
  * GROUP BY c_custkey. The reason for that is the functional dependency c_custkey => c_name, or, more practically
  * speaking, the fact that c_custkey is the primary key of the customer table.
- * This rule identifies cases where functionally dependent columns are given as group-by columns while their primary
- * key column(s) is a/are also group by column(s). It then replaces that column (e.g., c_name) with a dummy aggregate
- * function ANY(c_name). This reduces the grouping cost in the aggregate operator.
+ * This rule identifies cases where functionally dependent columns are given as group-by columns while the FD's
```

Comment:
Either introduce FD as an abbreviation or write it out as the abbreviation is only used twice